spec/models/course/reference_time_spec.rb

```diff
@@ -86,5 +86,40 @@
         end
       end
     end
+
+    describe '#reset_closing_reminders on assessment' do
+      let(:assessment) { create(:assessment, end_at: 2.days.from_now) }
+
+      before do
+        ActiveJob::Base.queue_adapter = :test
```

Comment:
Just wondering what do we need this for?

Reply:
If this line is not here, RSpec will complain that I set the queue adapter. Active Job tests need a specific `queue_adapter`.

I have since removed this and wrapped the entire test with `with_active_job_queue_adapter` to minimise code rewrite.